anylabeling/views/labeling/label_widget.py

```diff
@@ -2332,10 +2332,12 @@ def delete_file(self):
             os.remove(label_file)
             logger.info("Label file is removed: %s", label_file)
 
+            print(self.filename)
             item = self.file_list_widget.currentItem()
-            item.setCheckState(Qt.Unchecked)
-
+            if item != None:
```

Comment:
It's better to use `is not None`. Please check [this](https://note.nkmk.me/en/python-none-usage/#none-with-or).